corehq/apps/domain/templates/domain/pro_bono/domain.html

```diff
@@ -1,4 +1,9 @@
 {% extends "style/base_section.html" %}
+{% load hq_shared_tags %}
+
+{% block js %}{{ block.super }}
+    <script src="{% static 'domain/js/pro-bono.js' %}"></script>
+{% endblock %}
```

Comment:
there's probably a good reason that I'm not seeing, but why not put this in `pro_bono/base.html `?